share/models/meta.py

```diff
@@ -26,6 +26,7 @@ class Award(ShareObject):
     award = ShareURLField(blank=True)
     description = models.TextField(blank=True)
     url = ShareURLField(blank=True)
+    funder = ShareManyToManyField('Funder', through='ThroughAwardFunders')
```

Comment:
HyperTerm is really pretty neat. Especially with wow hyperpower mode. But tmux doesn't work in it.